iptables/_rule.sls

```diff
@@ -11,6 +11,9 @@ iptables_{{ chain_name }}_{{ rule_name }}:
   {%- endif %}
   - table: {{ rule.get('table', 'filter') }}
   - chain: {{ chain_name }}
+  {%- if rule.family is defined %}
+  - family: {{ rule.family }}
```

Comment:
H'm, well, I'll leave it to you if you want to give it a go then. Maybe I'm too paranoid but having such rules _without_ CIDRs is not the way I would want to manage my firewall. Not for ssh or any service really. A simple rule like that is also easy to repeat for both protocols; it's not like we're saving a lot of time and effort. And since it's more explicit it's harder to miss in review.